docs/browser_support.rst

```diff
@@ -0,0 +1,41 @@
+.. This Source Code Form is subject to the terms of the Mozilla Public
+.. License, v. 2.0. If a copy of the MPL was not distributed with this
+.. file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+===============
+Browser Support
+===============
+
+Mozilla websites should be tested in a broad range of browsers, though
```

Comment:
I realize that specific sites *might* be targeted at specific browsers/versions but I would encourage you to make a recommendation here on browser support. *Unless there are specific reasons you should look to fully support {...}, and these browsers can have degraded support {...}*